mvn_setup/tasks-parent/tasks-ui/src/core/Assignee/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { AvatarGroup, Box, ListItemText, InputAdornment, ListItem, Checkbox, Button, Avatar, List, TextField, ButtonProps, styled, ListItemTextProps } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+
+import Client from '@taskclient';
+
+import { TaskDescriptor } from 'taskclient/tasks-ctx-types';
+import { usePopover } from 'core/TaskTable/CellPopover';
+import { useAssignees } from 'taskclient/hooks';
+
+
+export const StyledButton = styled(Button)<ButtonProps>(() => ({
```

Comment:
This is broken encapsulation. You are exporting a component that is meant to be used inside of main component in the file.